src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
+    <NoWarn>$(NoWarn);CS1574;CS3016;CA5379;CA5384</NoWarn>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <RootNamespace></RootNamespace>
```

Comment:
Why is RootNamespace set to empty? I think we don't need this at all right?

Reply:
RootNamespace is here as empty because one or more of the things I have in VS decided that the library name was my default namespace, so it thought that AsnReader should be System.Formats.Asn1.System.Formats.Asn1.AsnReader, based off the file path from the project root.  This stops that from happening. :smile:

Reply:
that doesn't sound right, what happens if you remove the property entirely?

Reply:
Yeah, we don't do this on any other project, so this doesn't sound right.

Reply:
Microsoft.NET.Sdk defines it to default to the name of the csproj:

```xml
    <RootNamespace Condition=" '$(RootNamespace)' == '' ">$(MSBuildProjectName)</RootNamespace>
```

But if no other project is setting it I'll not be different.